test/e2e/performanceprofile/functests/13_llc/llc.go

```diff
@@ -102,58 +105,109 @@ var _ = Describe("[Performance] Last Level Cache", Label(string(label.OpenShift)
 
 			By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
 			profilesupdate.WaitForTuningUpdated(ctx, perfProfile)
+		}
+
+		// delete the machine config pool
+		err := testclient.Client.Delete(ctx, mc)
+		Expect(err).ToNot(HaveOccurred())
+
+		mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+		By("Waiting when mcp finishes updates")
+
+		mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+	})
+
+	Context("Configuration Tests", func() {
+
+		BeforeEach(func() {
+			if perfProfile.Annotations == nil || perfProfile.Annotations["kubeletconfig.experimental"] != llcPolicy {
+				testlog.Info("Enable align-cpus-by-uncorecache cpumanager policy")
+				perfProfile.Annotations = profileAnnotations
+
+				By("updating performance profile")
+				profiles.UpdateWithRetry(perfProfile)
 
-			for _, node := range workerRTNodes {
-				kubeletconfig, err := nodes.GetKubeletConfig(ctx, &node)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(kubeletconfig.CPUManagerPolicyOptions["prefer-align-cpus-by-uncorecache"]).To(Equal("true"))
+				By(fmt.Sprintf("Applying changes in performance profile and waiting until %s will start updating", poolName))
+				profilesupdate.WaitForTuningUpdating(ctx, perfProfile)
+
+				By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
+				profilesupdate.WaitForTuningUpdated(ctx, perfProfile)
 			}
 		})
 
-		It("[test_id: 77723] Removing the kubelet annotation should disable align-cpus-by-uncorecache policy", func() {
-
+		AfterEach(func() {
+			testlog.Infof("Remove align-cpus-by-uncorecache cpumanager policy")
```

Comment:
Same as my comment above: it is a policy option.

Reply:
Removed the AfterEach node in latest commit